framework-cloud/framework-cloud-api/src/main/java/org/kie/cloud/api/deployment/Deployment.java

```diff
@@ -39,6 +39,12 @@
      */
     void scale(int instances);
 
+    /**
+     * 
```

Comment:
It would be good to have also some extend description for future uses. e.g.:
```suggestion
     * Return deployment replicas count. Replicas count is change as deployment is scale up or down.
     *
```

Reply:
Changed